docs/docs/installation_guide/installation/pre-installation/oidc-providers.md

```diff
@@ -0,0 +1,127 @@
+---
+id: oidc-providers
+title: OIDC Providers Settings
+sidebar_label: OIDC Providers Settings
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section is for creating OIDC IdP client and creating Service Workbench root user in OIDC IdP.
+
+## Creating OIDC IdP Client
+
+[Cognito User Pool][cognito] is not supported in AWS China Regions. If you choose to deploy the solution in AWS China Regions, an OpenID Connector (OIDC) IdP client is required.
+
+You can use different kinds of OpenID Connector(OIDC) providers. This section introduces Option 1, Option 2 and Option 3.
+
+- (Option 1) [Authing][authing] is an example of a third-party authentication provider. 
+- (Option 2) [Keycloak on AWS](https://www.amazonaws.cn/en/solutions/keycloak-on-aws/) is a solution maintained by AWS and can serve as an authentication identity provider.
+- (Option 3) [Okta][okta] is one trusted platform to secure every identity, from customers to your workforce.
+
+Follow the steps below to create an OIDC client, and obtain the `client_id` and `issuer`. 
+
+### (Option 1) Authing.cn OIDC client
+
+1. Go to the [Authing console](https://console.authing.cn/console).
+2. Create a user pool if you don't have one.
+3. Select the user pool.
+4. On the left navigation bar, select **Self-built App** under **Applications**. 
+5. Click the **Create** button.
+6. Enter the **Application Name**, and **Subdomain**.
+7. Save the `App ID` (that is, `client_id`) and `Issuer` to a text file from Endpoint Information, which will be used later.
+    [![](../../../images/OIDC/endpoint-info.png)](../../../images/OIDC/endpoint-info.png)
+
+8. Update the `Login Callback URL` and `Logout Callback URL` to your IPC recorded domain name.
+    [![](../../../images/OIDC/authentication-configuration.png)](../../../images/OIDC/authentication-configuration.png)
+
+9. Set the Authorization Configuration.
+    [![](../../../images/OIDC/authorization-configuration.png)](../../../images/OIDC/authorization-configuration.png)
```

Comment:
not sure if it's a security best practice of configuring `client verification method for xxx token` to `none`

Reply:
Because we use "authorization_code' flow, user need to set `client verification method for xxx token` to `none`